packages/rich-text-react-renderer/src/index.tsx

```diff
@@ -80,6 +80,10 @@ export interface Options {
    * Text renderer
    */
   renderText?: RenderText;
+  /**
+   * Keep line breaks and multiple spaces
+   */
+  preserveFormatting?: boolean;
```

Comment:
Could we rename it to `preserveWhitespace` to be more clear? formatting could mean something else in the context of rich text